File: CSV/csv-tp.php

<?php

$list = array (
   array('first_name', 'last_name'),
   array('John', 'Doe'),
   array('Samus', 'Aran'),
   array('Gordon', 'Freeman'),
);

$fp = fopen('file.csv', 'w');

foreach ($list as $fields) {
    fputcsv($fp, $fields);
}

fclose($fp);
?>

File: CSV/csv-tp.py
import csv

with open('file.csv', 'w') as csvfile:
    fieldnames = ['first_name', 'last_name']
    
    writer = csv.DictWriter(csvfile, fieldnames=fieldnames)

    writer.writeheader()
    writer.writerow({'first_name': 'John', 'last_name': 'Doe'})
    writer.writerow({'first_name': 'Samus', 'last_name': 'Aran'})
    writer.writerow({'first_name': 'Gordon', 'last_name': 'Freeman'})
